src/test/java/com/puppycrawl/tools/checkstyle/internal/AllChecksTest.java

```diff
@@ -236,7 +237,8 @@ public class AllChecksTest extends AbstractModuleTestSupport {
         GOOGLE_TOKENS_IN_CONFIG_TO_IGNORE.put("IllegalTokenText", Stream.of(
                 // all other java tokens and text are allowed
                 "NUM_DOUBLE", "NUM_FLOAT", "NUM_INT", "NUM_LONG", "IDENT",
-                "COMMENT_CONTENT", "STRING_LITERAL", "CHAR_LITERAL", "TEXT_BLOCK_CONTENT")
+                "COMMENT_CONTENT", "STRING_LITERAL", "CHAR_LITERAL", "TEXT_BLOCK_CONTENT",
+                "STRING_TEMPLATE_CONTENT")
```

Comment:
This update is strange.
https://checkstyle.org/styleguides/google-java-style-20180523/javaguide.html#s2.3.2-special-escape-sequences
https://checkstyle.org/google_style.html#a2.3.2
https://github.com/checkstyle/checkstyle/blob/253430250fe784998d1b9d8a1c1b54c452751154/src/main/resources/google_checks.xml#L54

Why we ignoring such tokens, as they can have content that we need to validate ?

Reply:
We can definitely improve this in separate issue, just curious

Reply:
I do not remember reasoning, but we have added all new syntax to this collection. Maybe it was due to not being explicitly mentioned in Googles style guide.

Reply:
Issue created at https://github.com/checkstyle/checkstyle/issues/14291

Reply:
https://github.com/checkstyle/checkstyle/pull/14292